scripts/viz/param_sweep_viz.R

```diff
@@ -0,0 +1,74 @@
+# Pan-Cancer Variational Autoencoder
+# Gregory Way 2017
+#
+# Visualizing results of parameter sweep
+
+library(readr)
+library(ggplot2)
```

Comment:
also, unless you've come across sound reasons to use `ggplot2` namespace calls, I am going to leave loading in the library. 

I think that

```R
ggplot(subset_param, aes(x = train_epoch, y = loss)) +
    geom_line(aes(color = learning_rate, linetype = loss_type), size = 0.5) + 
    facet_grid(batch_size ~ epochs, scales = "free") + theme_bw() + 
    ggtitle(paste("kappa =", k))
```

is easier to digest than

```R
ggplot2::ggplot(subset_param, ggplot2::aes(x = train_epoch, y = loss)) +
    ggplot2::geom_line(ggplot2::aes(color = learning_rate, linetype = loss_type), size = 0.5) + 
    ggplot2::facet_grid(batch_size ~ epochs, scales = "free") + ggplot2::theme_bw() + 
    ggplot2::ggtitle(paste("kappa =", k))
```